accounts-service/src/main/java/jersey/rest/CreateAccountResource.java

```diff
@@ -0,0 +1,80 @@
+package jersey.rest;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import generator.Generator;
+import mybatis.account.AccountMapper;
+import pojo.Account;
+import regex.RegexValidator;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.net.HttpURLConnection;
+
+
+@Singleton
+@Path("/")
+public class CreateAccountResource {
+
+    private final AccountMapper accountMapper;
+
+    @Inject
+    public CreateAccountResource(AccountMapper accountMapper) {
+        this.accountMapper = accountMapper;
+    }
+
+    @POST
+    @Path("/create-account")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response createAccount(RequestAccountName requestAccountName) {
```

Comment:
If you want, Jersey allows to return an Account instance:
```suggestion
    public Account createAccount(RequestAccountName requestAccountName) {
```

The disadvantage is that you should throw exceptions instead of returning `Response.status([SOME_ERROR])`
Ask me about it

Reply:
Not doing anything about this